### PR TITLE
[Drupal] Add templates for each entityBundle

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR expands on our Drupal templating by adding a template for each (known) value for `entityBundle`, and loading each template dynamically during a for-loop.

`/drupal/test/index.html` seems to contain every type of `entityBundle`, so we can use that as an example. For now, the templates all do the same thing by just rendering `entityRendered` directly to the screen.

### CODE REVIEW

**Comment by ncksllvn** [comment]
@ahay-agile6

File: src/site/layouts/page.drupal.liquid [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% include "src/site/includes/header.html" %}
-<main>
+<main id="content">
```
Comment (ncksllvn) [inline src/site/layouts/page.drupal.liquid]: This is for the "Skip to content" link when you're tabbing on the site

File: src/site/layouts/page.drupal.liquid [hunk 2]
```diff
@@ -15,7 +15,9 @@
         {{ fieldIntroText }}
       </div>
       {% for block in fieldContentBlock %}
-        {{ block.entity.entityRendered }}
+        {% assign bundleComponent = "src/site/bundles/" | append: block.entity.entityBundle %}
+        {% assign bundleComponentWithExtension = bundleComponent | append: ".drupal.liquid" %}
+        {% include {{ bundleComponentWithExtension }} with block = block %}
```
Comment (ncksllvn) [inline src/site/layouts/page.drupal.liquid]: This is the dynamic import for new bundle templates. I'm sure there's a more elegant way to do this, but either way it does what it needs to do.

File: src/site/stages/build/options.js [hunk 3]
```diff
@@ -70,6 +71,7 @@ function applyDefaultOptions(options) {
       [`${includes}/**/*`]: '**/*.{md,html}',
       [`${components}/**/*`]: '**/*.{md,html}',
       [`${layouts}/**/*`]: '**/*.{md,html}',
+      [`${bundles}/**/*`]: '**/*.{md,html}',
```
Comment (ncksllvn) [inline src/site/stages/build/options.js]: Adds the new `bundle/` directory into the watch task.

File: src/site/includes/debug.drupal.liquid [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{% if buildtype == 'localhost' %}
+{% if buildtype != 'vagovprod' %}
```
Comment (ncksllvn) [inline src/site/includes/debug.drupal.liquid]: Per our discussion, the console will contain the Metalsmith data structure for all envs outside of prod.

File: src/site/layouts/page.drupal.liquid (hunk 2 above)
Comment (jbalboni) [reply src/site/layouts/page.drupal.liquid]: What happens if our template doesn't exist yet for a bundle? I can see that happening as they add new things in Drupal. Does the build process break?

Comment (ncksllvn) [reply src/site/layouts/page.drupal.liquid]: It doesn't break - it just skips that block as if there wasn't an include at all. I think we could add a filter to validate that the bundle template exists though if we do want to break it or log a warning?

Comment (jbalboni) [reply src/site/layouts/page.drupal.liquid]: I think a warning would be helpful, but we can do that later.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**